prow/plugins/milestone/milestone.go

```diff
@@ -35,7 +35,7 @@ const pluginName = "milestone"
 
 var (
 	milestoneRegex   = regexp.MustCompile(`(?m)^/milestone\s+(.+?)\s*$`)
-	mustBeSigLead    = "You must be a member of the [%s/%s](https://github.com/orgs/%s/teams/%s/members) github team to set the milestone."
+	mustBeAuthorized = "You must be a member of the [%s/%s](https://github.com/orgs/%s/teams/%s/members) github team to set the milestone. If you believe you should be able to issue the /milestone command, please contact your SIG Chairs/TLs and have them propose you as an additional delegate for this responsibility."
```

Comment:
I think we should add this information in `WhoCanUse` too, and replace the hardcoded "milestone maintainers" team with the value from the config.

https://github.com/kubernetes/test-infra/blob/49d1dc9a852745ec28f7d84f4edc70610c395077/prow/plugins/milestone/milestone.go#L75

Reply:
Can we configure this reponse? Not all repos using prow will use the SIG Chairs/TLs terminology.